module/EntraBeta/AdditionalFunctions/New-EntraBetaGlobalSecureAccessTenant.ps1

```diff
@@ -0,0 +1,15 @@
+# ------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+# ------------------------------------------------------------------------------
+function New-EntraBetaGlobalSecureAccessTenant {
```

Comment:
I am proposing we rename this command to `Enable-EntraBetaGlobalSecureAccess` since we are onboarding/activating/enabling GSA and not creating a new resource. I have intentionally left the word 'Tenant' in the naming since we can only activate GSA to a particular tenant - trivial.